Content.Server/GameObjects/Components/Interactable/ToolComponent.cs

```diff
@@ -122,18 +88,24 @@ public override void Initialize()
 
             _audioSystem = _entitySystemManager.GetEntitySystem<AudioSystem>();
             _interactionSystem = _entitySystemManager.GetEntitySystem<InteractionSystem>();
-            _toolSystem = _entitySystemManager.GetEntitySystem<ToolSystem>();
-
-            Owner.TryGetComponent(out _solutionComponent);
             Owner.TryGetComponent(out _spriteComponent);
         }
 
         public override void ExposeData(ObjectSerializer serializer)
         {
             base.ExposeData(serializer);
 
-            if(serializer.Reading)
-                _behavior = (Tool)serializer.ReadStringEnumKey("behavior");
+            if (serializer.Reading)
+            {
+                try
+                {
+                    _behavior = (Tool)serializer.ReadStringEnumKey("behavior");
+                }
+                catch
+                {
+                    // ignored
```

Comment:
Don't we have somewhere you could log this.

Reply:
Thing is, inherited components like the Welder might not use the behavior field... So I'm not sure if we should really log that hmm

Reply:
I changed this now to not rely on a try-catch block